### ISSUE TITLE
How to build lib 6.11 with current inform6 master

### ISSUE THREAD

**Issue by stormi** [DavidKinder/Inform6#294]
Part of the French IF community still uses inform 6. Our French libraries still rely on the 6.11 version of the inform libraries. Up to inform 6.42, we could still build games with these libraries, but the recent addition of verb name collision apparently highlights an issue in `infix.h` , so builds fail:

```
"inform6lib-6.11/infix.h", line 1053: Error:  Two different verb definitions refer to ";examine"
> Verb meta ';xo' ';examineo'
"inform6lib-6.11/infix.h", line 1055: Error:  Two different verb definitions refer to ";examine"
> Verb meta ';xs' ';examines'
```

We'd like to switch to 6.12.x someday, but it has an important impact on the French libs that no one has tackled yet, so for now we still rely on 6.11.

**Comment by erkyrath** [comment]
Yeah, I ran into this while testing. I applied this fix in my testing repository:

https://github.com/erkyrath/Inform6-Testing/commit/e24fd1a4fdbd6c6afb47455a5415370744ec488f

This just changes the Infix debug verbs `;EXAMINEO` and `;EXAMINES` to `;EXO` and `;EXS`.

**Comment by heasm66** [comment]
Just to clarify...

This error isn't limited to 6.11, you get the same complaint in 6.12.6.

`;examineo` and `;examines` are both 10 z-chars long so I can't see that these long versions of the verbs ever worked.

**Comment by erkyrath** [comment]
The outcome from that error was that the verbs *sometimes* worked right. I suspect one of them worked and the other one triggered some adjacent infix action, so it might not have been easy to notice.

Yeah, I'll add a bug report for the 6/12 library repo.

**Comment by stormi** [comment]
Is there a fixed 6.11 lib somewhere?

**Comment by erkyrath** [comment]
6.11 is not supported (and hasn't been since about 2005) so it won't be updated. But you can look at this directory: https://github.com/erkyrath/Inform6-Testing/tree/master/i6lib-611

Or just apply the change I linked above. (Replace `examineo` and `examines` in that file with `exo` and `exs`.)
